src/manifolds/Euclidean.jl

```diff
@@ -10,219 +10,259 @@ Euclidean vector space $\mathbb R^n$.
 
     Euclidean(n)
 
-generates the $n$-dimensional vector space $\mathbb R^n$.
+Generate the $n$-dimensional vector space $\mathbb R^n$.
 
     Euclidean(n₁,n₂,...,nᵢ; field=ℝ)
 
-generates the (2*)$n_1n_2\cdot\ldots n_i$-dimensional vector space $\mathbb F^{n_1, n_2, \ldots, n_i}$, whose
+Generate the (2*)$n_1n_2\cdot\ldots n_i$-dimensional vector space
```

Comment:
I'm not sure if the `(2*)` part is clear enough here. Maybe we should write the dimension at the end of this docstring, after `field` is mentioned?